README.md

```diff
@@ -4,25 +4,10 @@ phpspec2-expect
 Installation
 ============
 
-Add `"bossa/phpspec2-expect": "*"` to `composer.json`
-
-```json
-{
-    "require-dev": {
-         "phpspec/phpspec": "dev-master",
-         "bossa/phpspec2-expect": "*"
-    },
-    "config": {
-        "bin-dir": "bin"
-    },
-    "minimum-stability": "dev",
-    "autoload": {
-        "psr-0": {
-            "spec": "",
-            "": "src"
-        }
-    }
-}
+Install it using `composer require` command:
```

Comment:
Thanks. English is not my native language so forget to add those articles too often